archs/x86/text.xml

```diff
@@ -28,9 +28,6 @@ The following is a list of rules and expectations for members of the x86 team:
 </p>
 
 <ul>
-  <li>
-    Assist/resolve 4 bugs a month that are assigned to the team
-  </li>
   <li>
     Be present in our IRC channel, <c>#gentoo-x86/irc.libera.chat</c>
```

Comment:
Drop this too, please. It's now a redirect to #gentoo.

Reply:
done